File: backend/ambevtech/src/main/java/com/ambevtech/core/entity/dto/DadosTempoDTO.java
package com.ambevtech.core.entity.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.io.Serializable;
import java.util.List;

@Getter
@Setter
@AllArgsConstructor
public class DadosTempoDTO implements Serializable {

//    @SerializedName("daily")
    private List<TempoCidadeDTO> daily;

    private List<AlertasTempoDTO> alerts;

}

File: frontend/src/app/model/DadosTempoDTO.ts
import { AlertasTempoDTO } from "./AlertasTempoDTO";
import { TempoAtualDTO } from "./TempoAtualDTO";
import { TempoCidadeDTO } from "./TempoCidadeDTO";

export class DadosTempoDTO {

    current: TempoAtualDTO;
    daily: Array<TempoCidadeDTO>;
    alerts: Array<AlertasTempoDTO>;

    constructor(){
        this.daily = new Array<TempoCidadeDTO>();
        this.alerts = new Array<AlertasTempoDTO>();
    }

}